packages/components/addon/components/hds/table/index.js

```diff
@@ -89,6 +110,11 @@ export default class HdsTableIndexComponent extends Component {
       classes.push(`hds-table--density-${this.density}`);
     }
 
+    // add a class based on the @valign argument
+    if (this.valign) {
+      classes.push(`hds-table--valign-${this.valign}`);
+    }
```

Comment:
since we have a default value for `valign` (similarly for `density`) the conditional statements are always going to be `true`, so we can
```suggestion
    classes.push(`hds-table--valign-${this.valign}`);
```